xcell/mappers/mapper_DELS.py

```diff
@@ -89,6 +122,9 @@ def _get_lorentzian(self, zz):
         return 1./(1+((zz[:, None]-zz[None, :]-m)/s)**2/(2*a))**a
 
     def _get_nz(self):
+        # Builds the redshift distributions of \
```

Comment:
Why comments instead of docstring? (same for the other functions here and in other mappers)